### ISSUE TITLE
error: implicit declaration of function 'openpty'

### ISSUE THREAD

**Issue by ryandesign** [toddr/IO-Tty#25]
Building IO:Tty 1.14 on macOS with Xcode 12 beta and perl 5.28.3 fails:

```
Tty.xs:530:12: error: implicit declaration of function 'openpty' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
            ret = openpty(ptyfd, ttyfd, NULL, NULL, NULL);
                  ^
```

What worked for me was to revert b59e8e2b6f001a135d49cb8f10a9ca4d49a1d5c5.

**Comment by Hugmeir** [comment]
The underlaying issue here is _great_.

OSX provides the `openpty` declaration via `util.h`.
Perl itself provides a completely unrelated `util.h`.

When compiling XS modules, Perl puts its includes first, so its util.h is picked up and used, but that leaves the system util.h unreachable.

**Comment by toddr** [comment]
Fixed by #23 ?
